src/main/resources/templates/upload.html

```
@@ -9,41 +9,36 @@
+                    <button th:data-file="${file.getS3Key()}"
+                            onclick="downloadFile(this.getAttribute('data-file'))"
+                            class="button">Download</button>
+                </td>
+            </tr>
             </tbody>
         </table>
     </section>
+    <p><a th:href="@{/user}"></a></p>
+</div>
+
 </main>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </main> ]

(tag-pair)